frontend/src/components/RepositoryForm.vue

```diff
@@ -168,15 +168,15 @@
         >
       </div>
       <div class="mt-1 textinfolabel">
-        <template v-if="isProjectSchemaChangeTypeDDL">
+        <template v-if="isDev && !isProjectSchemaChangeTypeDDL">
```

Comment:
This may not be needed as we will get isProjectSchemaChangeTypeDDL only when line 76 is open.

Reply:
They are not in a div. And this is a description text for SDL schema path template, so I'd like to add `isDev` && `isNotDDL`.

Reply:
Either way works, trying to minimize the change. !isProjectSchemaChangeTypeDDL cannot happen if !isDev.

Reply:
Yeah, it's a same commit as the cherry-pick one. In the release/1.4.0 branch, we need to use `isDev` flag to prevent the `schemaChangeType` is empty.